src/fmeval/reporting/constants.py

```diff
@@ -168,6 +170,8 @@ class ListType(Enum):
     F1_SCORE: "Numerical score between 0 (worst) and 1 (best). F1-score is the harmonic mean of precision and recall. It is computed as follows:  precision = true positives / (true positives + false positives) and recall = true positives / (true positives + false negatives). Then F1 = 2 (precision * recall)/(precision + recall) .",
     EXACT_MATCH_SCORE: "An exact match score is a binary score where 1 indicates the model output and answer match exactly and 0 indicates otherwise.",
     QUASI_EXACT_MATCH_SCORE: "Similar as above, but both model output and answer are normalised first by removing any articles and punctuation. E.g., 1 also for predicted answers “Antarctica.” or “the Antarctica” .",
+    PRECISION: "The precision score is the fraction of words in the model output that are also found in the target output.",
```

Comment:
Suggestion: Let us call this `PRECISION_OVER_WORDS`. Right now, we have `PRECISION` under QA accuracy and `PRECISION_SCORE` under classification accuracy. The two metrics are quite different -- one is computed over words in a _single_ input, the other is computed over _all_ the inputs. Let us reflect the differences in the names to avoid confusion. 

Same for `RECALL`.